org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/tests/test.js

```diff
@@ -1,13 +1,10 @@
 var test = require('ava');
 var OpenT2T = require('opent2t').OpenT2T;
-var config = require('./testConfig');
-
-console.log("Config:");
-console.log(JSON.stringify(config, null, 2));
-
 var translatorPath = require('path').join(__dirname, '..');
+var runBinarySwitchTests = require('opent2t-device-binaryswitch/binarySwitchTests');
+var config = require('./testConfig');

```

Comment:
and everyone of them in any folder

---
In reply to: [90384226](https://github.com/openT2T/translators/pull/135#discussion_r90384226) [](ancestors = 90384226)